source/ore/routes/deploy-version.rst

```diff
@@ -0,0 +1,57 @@
+========================
+Deploy Project Version
+========================
+
+**POST /api/v1/projects/:pluginId/versions/:version**
+
+Deploys a new version for the specified project. The body should be sent as ``multipart/form-data`` with the fields shown below. Returns a JSON representation of the deployed version.
+
++-------------+------------------------------+------------------------------------------------------------------------------------------------+
```

Comment:
Does the table really need to be this wide in raw format? Our preference is 120 character wide or lower. (Guideline 13 in https://docs.spongepowered.org/stable/en/contributing/spongedocs.html). It's easier to edit.